# This represents the parameters of transformation system of a determined DMP.
bool initialized
int32 trans_sys_id
float32 k_gain
float32 d_gain
lwr/Model lwr_model
